admin/client/dist/js/AddToCampaignForm.js

```diff
@@ -22,7 +22,7 @@
   }
 
   _jQuery2.default.entwine('ss', function ($) {
-    $('.add-to-campaign-action, #add-to-campaign__action').entwine({
+    $('#add-to-campaign__dialog .add-to-campaign-action,' + '.cms-content-actions .add-to-campaign-action,' + '#add-to-campaign__action').entwine({
```

Comment:
yeah, linting was complaining about the src file going over 100 characters long for a line, so I broke each selector down to their own line...

I could do `$('select').add('another').add('more')` syntax instead, if that's preferred
it just looks funny with the string concat in the dist file